fastlane/Fastfile

```diff
@@ -245,6 +246,58 @@ platform :ios do
     )
   end
 
+  lane :generate_changelog do
+
+  end
```

Comment:
looks like this one wasn't implemented / used?

Reply:
Ah, whoops! Forgot I did that 😛